### ISSUE TITLE
Production Guide: Does not work "sudo -iu mastodon"?

### ISSUE THREAD

**Issue by h-izumi** [tootsuite/documentation#316]
Since #306, Production Guide has guide to `useradd`.

```shell
sudo useradd --system --user-group --shell /bin/false --create-home --home /home/mastodon mastodon
```

This sets `/bin/false` as shell, so "sudo -iu mastodon" takes no effect.

Is this guide correct?

**Comment by wxcafe** [comment]
That's.... actually a good point. I'm really sorry, this is my fault. I'm gonna fix it right now.

**Comment by wxcafe** [comment]
fixed in 39fe30df48c36a22b4c0461233ccf6cde421c273 (I checked on a server) sorry for that
